TestSwitchApi/TestSwitchApi/Services/CookieService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Net;
+using Microsoft.AspNetCore.Http;
+
+namespace TestSwitchApi.Services
+{
+     public class CookieService : ICookieService
+     {
+         public void MakeNewLoginCookie(Guid sessionId, HttpContext context)
+         {
+             var cookieOptions = new CookieOptions
+             {
+                 Expires = new DateTimeOffset(DateTime.Now.AddDays(1)),
+                 Domain = "https://testswitch-admin-staging.herokuapp.com",
```

Comment:
This will need to be an environment variable - otherwise it won't work in production :(

Reply:
Is this what you mean? Not great with environment variables